tests/test_read_configs_include.py

```diff
@@ -43,27 +43,19 @@ class TestReadConfigs:
     default_all_conf_order: list[str] = [
         "./config/proxy",
         "./config/canarie-api",
+        "./config/geoserver",
```

Comment:
Any reason GeoServer had to be moved up?

Reply:
That's the order that components are loaded now by default. Previously phoenix had a bunch of dependencies that were loaded first so the load order looked quite different.